### PR TITLE
Adding Artifactory reference architecture diagram

### PR DESCRIPTION
**Pull request by jayyeras**

### CODE REVIEW

File: modules/artifactory/readme.adoc [hunk 1]
```diff
@@ -1,8 +1,21 @@
 = Artifactory
```
Comment (christopherhein) [inline modules/artifactory/readme.adoc]: Can we make this `== Artifactory` and then add `= Digital Modernization` above this.

so it should be:

```
= Digital Modernization

== Artifactory
```

File: modules/artifactory/readme.adoc [hunk 2]
```diff
@@ -1,8 +1,21 @@
...
-fatback shank. Chicken venison cow, leberkas ball tip bacon porchetta ground
-round salami hamburger doner meatloaf alcatra frankfurter beef.
+****
+*Expected Outcome:*
+
+* Understanding of a binary repository manager
+* Deploy JFrog Artifactory
+
+*Lab Requirements:*
+None
```
Comment (christopherhein) [inline modules/artifactory/readme.adoc]: Are there none?

File: modules/continuous-integration/readme.adoc [hunk 3]
```diff
@@ -0,0 +1,8 @@
+= Continuous Integration (CI)
```
Comment (christopherhein) [inline modules/continuous-integration/readme.adoc]: Same as the other module, just add the `= Digital Modernization` above and make this an h2 `==`

**Comment by jayyeras** [comment]
Making changes and cleaning up the PR as well.

**Pull request closed without merging**